docs/conf.py

```diff
@@ -59,10 +59,10 @@
 #
 # The short X.Y version.
 try:
-    import scrapy
-    version = '.'.join(map(str, scrapy.version_info[:2]))
-    release = scrapy.__version__
-except ImportError:
+    version_path = Path(__file__).parent.absolute().parent.joinpath("scrapy/VERSION")
```

Comment:
This should be safe I think (https://github.com/scrapy/scrapy/blob/2.0.1/setup.py#L6-L7)

Reply:
Was there something wrong with the previous implementation? `scrapy.__version__` seems cleaner to me.

Reply:
Indeed, I change it because it wasn't working for me, but I just found it was because I didn't have Scrapy installed in the virtualenv for the docs. I'll revert it, thanks for noticing.